libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/history/ReplayEvents.kt

```diff
@@ -2,26 +2,59 @@ package com.mapbox.navigation.core.replay.history
 
 import com.google.gson.annotations.SerializedName
 
+/**
+ * Replay event that fetched from [ReplayHistoryDTO]
+ *
+ * @param events Assumes chronological order, index 0 moves to [events.size] over time.
+ */
 data class ReplayEvents(
-    // Assumes chronological order, index 0 moves to events.size over time.
     val events: List<ReplayEventBase>
 )
 
+/**
+ * Base interface event for ReplayEvent
+ *
+ * @property eventTimestamp timestamp of event milliseconds
+ * @see [ReplayHistoryLocationEngine]
+ */
 interface ReplayEventBase {
     val eventTimestamp: Double
 }
 
+/**
+ * todo docs
+ * @param eventTimestamp timestamp of event milliseconds
+ * @param timestamp Double todo docs
```

Comment:
```suggestion
 * The getStatus event from history. This may be deprecated soon but can
 * be useful for comparing versions.
 * 
 * @param eventTimestamp timestamp of event in seconds
 * @param timestamp timestamp of event in seconds
```

Reply:
@kmadsen what is different between `eventTimestamp` and `timestamp` params? would be great to write it down in docs

Reply:
There's no difference. could delete `timestamp`

Reply:
@kmadsen thanks. I wouldn't like to remove it through current PR, gonna merge it as is. Shall I create issue for it?

Reply:
I can follow up, no problem